src/content/docs/en/reference/experimental-flags/serialized-configuration.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Experimental serialized configuration
+sidebar:
+  label: Serialized configuration
+i18nReady: true
+---
+
+import Since from '~/components/Since.astro';
+
+<p>
+
+  **Type:** `boolean`<br />
+  **Default:** `false`<br />
+
+  <Since v="5.2.0" />
+</p>
+
+This feature allows access to the `astro:config` virtual module which exposes a non-exhaustive, serializable, type-safe version of the Astro configuration through two sub-modules.
+
+For a complete overview, and to give feedback on this experimental API, see the [Serialized Manifest RFC](https://github.com/withastro/roadmap/blob/feat/serialised-config/proposals/0051-serialized-manifest.md).
+
+To enable this virtual module, add the `experimental.serializeManifest` feature flag to your Astro config:
+
+```js title="astro.config.mjs" ins={3-5}
+import {defineConfig} from "astro/config"
+export default defineConfig({
+  experimental: {
+    serializeManifest: true
+  }
+})
+```
+
+## Submodules
+
+The virtual module exposes two submodules for accessing different subsets of your configuration values. This protects your information by only making some data available to the client, while keeping other data available only on the server.
+
+## `astro:config/client`
+
+It exposes information that are **safe** to expose to the client (browser), hence it can be used in any instance of the application. It exposes information such as `trailingSlash`, `build.format`, `i18n`, and more.
```

Comment:
```suggestion
It exposes information that are **safe** to expose to the client (browser), hence it can be used in any instance of the application. It exposes information such as `trailingSlash`, `build.format`, `i18n`, and more.

This is a developing feature. For a full, up-to-date list of the configuration values available from `astro:config/client`, please see [the proposed API from the feature proposal](hhttps://github.com/withastro/roadmap/blob/feat/serialised-config/proposals/0051-serialized-manifest.md#proposed-apis:~:text=astro%3Aconfig%2Fclient%2C%20exported%20bindings%3A)
```